src/garage/torch/policies/context_conditioned_policy.py

```diff
@@ -0,0 +1,268 @@
+# pylint: disable=attribute-defined-outside-init, protected-access
```

Comment:
can you explain why each of these is necessary?

Reply:
also, please only use these disables at the callsite, rather than for the entire file